src/api/python/cvc5.pxi

```diff
@@ -1584,6 +1598,35 @@ cdef class Term:
         term.cterm = self.cterm.iteTerm(then_t.cterm, else_t.cterm)
         return term
 
+    def isInt32(self):
+        return self.cterm.isInt32()
+
+    def getInt32(self):
+        return self.cterm.getInt32()
+    
+    def isUInt32(self):
+        return self.cterm.isUInt32()
+    
+    def getUInt32(self):
+        return self.cterm.getUInt32()
+    
+    def isInt64(self):
+        return self.cterm.isInt64()
+    
+    def getInt64(self):
+        return self.cterm.getInt64()
+    
+    def isUInt64(self):
+        return self.cterm.isUInt64()
+    
+    def getUInt64(self):
+        return self.cterm.getUInt64()
```

Comment:
I added them to be able to translate this test:
https://github.com/cvc5/cvc5/blob/cd1f1c3f308b67fc4b8f006196e5bc1f366cc10d/test/unit/api/term_black.cpp#L667

should I just remove the corresponding test from the python unit test?

Reply:
The whole idea of the `isX()` / `getX()` methods was to translate terms to native types as faithfully as possible. As python integers can be arbitrarily large anyway, there is no point in exposing them. I'd suggest something like that:
```python
def getInteger(self):
  if self.cterm.isInt64():
    return self.cterm.getInt64()
  return int(self.cterm.getInteger())
```

Reply:
`getInteger()` in the cpp API returns a string:
https://github.com/cvc5/cvc5/blob/master/src/api/cpp/cvc5.h#L1259

So why don't I just return a string in the python API as well (and let the user cast with `int(.)` if they want to)?
This is what I did in ed57020, ket me know if it is better to do something else.

Reply:
The Python API already has `toPythonObject` for coverting to Python native objects: https://github.com/cvc5/cvc5/blob/master/src/api/python/cvc5.pxi#L1587

Reply:
so does it still make sense to add `getInteger` that will return a string (like in the cpp API)? I think it does, as `toPythonObject` will return a python integer, and also to be consistent with the cpp API.

Reply:
Honestly, I don't think it makes much sense. If you need a string in python just use `str(...)`.

Reply:
I'm somewhat confused about the way this is going. In the C++ API, we return have `getInteger` return a string because this is the best type we have in (standard) C++ for arbitrarily large integers. In Python, however, an int can hold integers, and IMHO we should use it.
I find this "be consistent with the C++ API" a weak argument. We should be for function names (and even there, I'm not sure), but I'd definitely use return types that make sense for the language at hand. (for example, I'd also suggest to use `BitInteger` in Java instead of a string...)